### PR TITLE
feat: update org name

### PR DESCRIPTION
**Pull request by gitanjli525**
## Type of Change

- [ ] Bugfix
- [x] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description

Allowing Org admins to update org name

image
image
image

## How did you test it?

Click on pencil icon on Org switch dropdown box
Enter new org name
org name gets updated in dropdown

## Where to test it?

- [x] INTEG
- [x] SANDBOX
- [ ] PROD

## Checklist

- [x] I ran `npm run re:build`
- [x] I reviewed submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

**Comment by sarojtailor** [comment]
Will clicking on the pencil icon open the modal?  Also, can we have the pencil icon with the org name and not the title

**Comment by gitanjli525** [comment]
> Will clicking on the pencil icon open the modal? Also, can we have the pencil icon with the org name and not the title

Yes @sarojtailor it opens the modal, okay will check how it looks with pencil near org name

**Review by sagarnaikjuspay — DISMISSED** [review]

File: src/screens/OMPSwitch/OMPSwitchHelper.res [hunk 1]
```diff
@@ -177,3 +180,81 @@ let generateDropdownOptions: array<OMPSwitchTypes.ompListTypes> => array<
...
+      }
+
+      if errorMessage->isNonEmptyString {
+        Dict.set(errors, "organization_name", errorMessage->JSON.Encode.string)
+      }
+
+      errors->JSON.Encode.object
+    }
+
+    let businessName = FormRenderer.makeFieldInfo(
```
Comment (JeevaRamu0104) [inline src/screens/OMPSwitch/OMPSwitchHelper.res:220]: change this to orgName

File: src/screens/OMPSwitch/OMPSwitchHelper.res [hunk 2]
```diff
@@ -177,3 +180,81 @@ let generateDropdownOptions: array<OMPSwitchTypes.ompListTypes> => array<
...
+          <div className="flex flex-col gap-12 h-full w-full">
+            <FormRenderer.DesktopRow>
+              <FormRenderer.FieldRenderer
+                fieldWrapperClass="w-full"
+                field={businessName}
+                labelClass="!text-black font-medium !-ml-[0.5px]"
+              />
+            </FormRenderer.DesktopRow>
+            <div className="flex justify-end w-full pr-5 pb-3">
+              <FormRenderer.SubmitButton text="Submit changes" buttonSize={Small} />
```
Comment (JeevaRamu0104) [inline src/screens/OMPSwitch/OMPSwitchHelper.res:253]: please check the button state.It should be loading

File: src/screens/OMPSwitch/OMPSwitchHelper.res (hunk 1 above)
Comment (gitanjli525) [reply src/screens/OMPSwitch/OMPSwitchHelper.res]: okay

File: src/screens/OMPSwitch/OMPSwitchHelper.res (hunk 2 above)
Comment (gitanjli525) [reply src/screens/OMPSwitch/OMPSwitchHelper.res]: okay

**Review by JeevaRamu0104 — APPROVED** [review]

**Review by sagarnaikjuspay — APPROVED** [review]

**Pull request merged**